guides/covariates_tfms.md

```diff
@@ -0,0 +1,289 @@
+# In-depth look at Covariates in Torch Forecasting Models
+This document was written for darts version 0.15.0.
+
+We assume that you already know about covariates in Darts. If you're new to the topic we recommend you to read darts/guides/covariates.md first.
+
+## Summary - TL;DR
+In Darts, **Torch Forecasting Models (TFMs)** are broadly speaking "machine learning based" models, which denote PyTorch-based (deep learning) models.
+
+TFMs train and predict on fixed-length chunks (sub-samples) extracted from your input data. Each chunk contains an input chunk - resembling the sample's past - and an output chunk - the sample's future. The length of these chunks has to be specified at model creation with parameters `input_chunk_length` and `output_chunk_length`.
+
+```
+# model that looks 7 time steps back (past) and 1 time step ahead (future)
+model = SomeTorchForecastingModel(input_chunk_length=7, 
+                                  output_chunk_length=1, 
+                                  **model_kwargs)
+```
+
+All TFMs can be trained on single or multiple `target` series and, depending on their covariate support, `past_covariates` and / or `future_covariates`.
+
+```
+# fit the model on a single target series with optional past and / or future covariates
+model.fit(target, 
+          past_covariates=past_covariates, 
+          future_covariates=future_covariates)
+```
+
+You can produce forecasts for any input `target` TimeSeries (also unobserved targets) as long as it contains `input_chunk_length` time steps.
```

Comment:
```suggestion
You can produce forecasts for any input `target` TimeSeries or for several targets given as a sequence of TimeSeries. This will also work on series that have not been seen during training, but the series must contain `input_chunk_length` time steps.
```